examples/erc20/DAI/DAI.xml

```diff
@@ -2,25 +2,90 @@
 <!DOCTYPE token  [
     <!ENTITY style SYSTEM "shared.css">
     <!ENTITY DAI-bridge.en SYSTEM "dai-bridge.en.js">
+    <!ENTITY item-view-ownerApproved.en SYSTEM "item-view-ownerApproved.en.js">
+    <!ENTITY item-view-minted.en SYSTEM "item-view-minted.en.js">
+    <!ENTITY item-view-redeemed.en SYSTEM "item-view-redeemed.en.js">
     <!ENTITY bestRates.en SYSTEM "../BestRates/bestRates.en.js">
+    <!ENTITY supply.en SYSTEM "../Compound/supply.en.js">
+    <!ENTITY withdraw.en SYSTEM "../Compound/withdraw.en.js">
+    <!ENTITY enable.en SYSTEM "../Compound/enable.en.js">
     ]>
 <ts:token xmlns:ts="http://tokenscript.org/2020/06/tokenscript"
           xmlns:ethereum="urn:ethereum:constantinople"
           xmlns:xhtml="http://www.w3.org/1999/xhtml"
           xmlns:xml="http://www.w3.org/XML/1998/namespace"
+          xmlns:asnx="urn:ietf:params:xml:ns:asnx"
           xsi:schemaLocation="http://tokenscript.org/2020/06/tokenscript http://tokenscript.org/2020/06/tokenscript.xsd"
-xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           custodian="false"
 >
+
+    <asnx:module name="ERC20-Events">
+        <namedType name="Approval">
+            <type>
+                <sequence>
+                    <element name="owner" ethereum:type="address" ethereum:indexed="true"/>
+                    <element name="spender" ethereum:type="address" ethereum:indexed="true"/>
+                    <element name="amount" ethereum:type="uint256" ethereum:indexed="false"/>
+                </sequence>
+            </type>
+        </namedType>
+        <namedType name="Mint">
+            <type>
+                <sequence>
+                    <element name="minter" ethereum:type="address" ethereum:indexed="false"/>
```

Comment:
Is it correct that every field is not indexed? Then we can't build a filter?

Reply:
Replaced with aDAI